ra/ra_cfg/fsp_cfg/r_dtc_cfg.h

```diff
@@ -0,0 +1,40 @@
+/*

```

Comment:
I think these are based on files generated by FSP tools.
How about creating a module/hal_renesas_fsp directory on the zephyr side and placing it there?
(Also, I think it would be easier to manage if only fsp releases were included)

Reply:
@soburi Thanks for your feedback.
I agree with you but a second module looks like an overkill to me. I would prefer if we followed the following structure:
```
.
├── fsp
│   ├── ra
│   └── any other family
├── smartbond
└── zephyr
    ├── blobs
    ├── fsp_cfg
    │  	├── ra
    │  	└── any other family
    └── any other folder/file that will help us use FSP with Zephyr
```
@thaoluonguw @KhiemNguyenT @ioannis-karachalios Your thoughts?

Reply:
I do agree that a second module would be an overkill.

Reply:
@thaoluonguw @ioannis-karachalios 

I got it. Let's go with the method you suggested.